cargo/cargo_bootstrap.bzl

```diff
@@ -239,6 +241,10 @@ cargo_bootstrap_repository = repository_rule(
             ],
             default = "release",
         ),
+        "cargo_config": attr.label(
+            doc = "The cargo config file.",
+            allow_single_file = [".cfg"],
```

Comment:
Aren't cargo configuration files expected to be named either `config` or `config.toml`? Where does `.cfg` come from? Sorry if I'm overlooking something in https://doc.rust-lang.org/cargo/reference/config.html

Reply:
I'm just going to remove the extension restriction. I don't see any good reason for it.